### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Scotty-Brown** [Scotty-Brown/beyond-borders#3]
- You must be able to tab through your app and use it without a mouse
- Your app must still be usable when tested with a colorblind extension
- You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
- Your HTML must be written semantically and should use ARIA tags (ONLY if needed / appropriate)
